eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/student/student.controller.js

```diff
@@ -0,0 +1,204 @@
+﻿(function (angular) {
+	angular.module('equizModule').controller('StudentController', StudentController);

```

Comment:
Use the following convention:


angular
.module('equizModule')
.controller('StudentController', StudentController);